### ISSUE TITLE
nnlm with vector response

### ISSUE THREAD

**Issue by RamilNurtdinov** [linxihui/NNLM#26]
Hi

I am trying to run nnlm functions.
It is written that 
y : a vector or matrix of responses.
However I cannot run the function with single column response.

Your example:
x <- matrix(runif(50*20), 50, 20);
beta <- matrix(rexp(20*2), 20, 2);
y <- x %*% beta + 0.1*matrix(runif(50*2), 50, 2);

beta.hat <- nnlm(x, y, loss = 'mkl');
This works perfectly

However
beta.hat <- nnlm(x, y[,1], loss = 'mkl');
Error in colSums(sol$coefficients) :
  'x' must be an array of at least two dimensions

I can make a temporary solution that works somehow
z = cbind(y[,1], y[,1])
beta.hat <- nnlm(x, z, loss = 'mkl');
